models.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+from flask_sqlalchemy import SQLAlchemy
+db = SQLAlchemy()
+
+class ACMRequest(db.Model):
+    __tablename__ = "acm_requests"
+    id = db.Column(db.Integer, primary_key=True)
+    body = db.Column(db.String(100))
```

Comment:
Do we think 100 characters is going to be sufficient? (That's less than a tweet... 😛)